### ISSUE TITLE
Importing OpenIJTIFF

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#561]
@manerotoni @tibuch @sebgoti @grinic 

I find this now the easiest to have the OpenIJTIFF available in Jupyter:

<img width="823" alt="image" src="https://github.com/NEUBIAS/training-resources/assets/2157566/751dab63-4d06-4f7f-92df-23193b8983ef">

**Comment by manerotoni** [comment]
Hi @tischi, 
was it meant that you change the module code of the master branch. 
People are a little puzzled what the line stands for. I don't think it is a good idea. 
We should install the OpenIJTIFF in the conda environment, this is the cleanest way.

We have now a bunch of exercise with ` append/tischer`

**Comment by tischi** [comment]
Yes, sorry. We should remove this again. Problem is that when I open a python file with jupytext it **auto-saves** my changes and when I am at the same time on master this is what happens....not sure yet what would be the best way for me in practice....

I think in general we still need to find a better way to import the OpenIJTIFF.

**Comment by tischi** [comment]
Another option would be to leave this cell in and add a comment:

`# Please adapt the folder to where you have OpenIJTIFF.py on your computer` 

Personally I would find that more convenient....

**Comment by tischi** [comment]
> What is possible to do is that you add openIJTIFF to your environment. 

That sounds great! How does one do this?

**Comment by manerotoni** [comment]
The other way would be to packege the script and then install it. 
It works but it is a little bit of additional editing (setup tool, init file etc.)

**Comment by grinic** [comment]
For now, I am following exactly the workflow that is highlighted in "tool installation": I copy the `OpenIJTIFF.py` file in a directory where I also copy all the .py activities.
The directory is outside of the repository folder, this prevents the autosaving from changing the master.

A "simple" but probably not very recommended way is to copy `OpenIJTIFF.py` in the `site-packages` directory of the python environment:
`miniconda3\envs\skimage-napari-tutorial\Lib\site-packages`

**Comment by tischi** [comment]
I think adding it to the PYTHONPATH is the best for my use case.

Should we still leave the `import sys` stuff in the current scripts or remove it?
